perl/CPAN-ONLY-CHANGES

```diff
@@ -0,0 +1,15 @@
+This file contains changes in the CPAN-only releases of this module. Its entries
```

Comment:
As mentioned above it would simplify our process a lot if we don't have to maintain this file.